web/src/feedback_widget.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
Why is this disabled?

Reply:
My bad I used this while on dev when I was still figuring on how to fix this es lint rule `@typescript-eslint/unbound-method` but its now fixed I just forgot to remove it